testing/test_faulthandler.py

```diff
@@ -49,8 +51,19 @@ def test_disabled():
     assert result.ret == 0
 
 
-@pytest.mark.parametrize("enabled", [True, False])
-def test_timeout(testdir, enabled):
+@pytest.mark.parametrize(
+    "enabled",
+    [
+        pytest.param(
+            True,
+            marks=pytest.mark.skipif(
+                sys.platform.startswith("win"), reason="issue #7022",
```

Comment:
Please provide a "real" reason.  Issue numbers have to be looked up etc.  You can sill have it in parenthesis there then.